File: 3515-find-if-digit-game-can-be-won/find-if-digit-game-can-be-won.cpp
class Solution {
public:
    bool canAliceWin(vector<int>& nums) {
        int singleSum=0,doubleSum=0;
        for(int i=0; i<nums.size(); i++)
        {
            if(nums[i]>9)
            {
                doubleSum += nums[i];
            }else{
                singleSum += nums[i];
            }
        }
        if(singleSum == doubleSum){
            return false;
        }
        return true;
    }
};

File: 3515-find-if-digit-game-can-be-won/find-if-digit-game-can-be-won.py
class Solution:
    def canAliceWin(self, nums: List[int]) -> bool:
        singleDigit = 0
        doubleDigit = 0
        for val in nums:
            if val>9:
                doubleDigit +=val
            else:
                singleDigit += val
        if doubleDigit == singleDigit:
            return False
        return True